src/leetCodeExecutor.ts

```diff
@@ -99,7 +99,7 @@ class LeetCodeExecutor implements Disposable {
         const filePath: string = path.join(outDir, fileName);
 
         if (!await fse.pathExists(filePath)) {
-            const codeTemplate: string = await this.executeCommandWithProgressEx("Fetching problem data...", this.nodeExecutable, [await this.getLeetCodeBinaryPath(), "show", problemNode.id, "-cx", "-l", language]);
+            const codeTemplate: string = await this.executeCommandWithProgressEx("Fetching problem data...", this.nodeExecutable, [await this.getLeetCodeBinaryPath(), "show", problemNode.id, "-c", "-l", language]);
```

Comment:
Since we can view problem description in preview webview, description comment in solution file (-x) is no longer needed.

Reply:
My suggestion is that we keep it as `-cx` for now. And open an issue to see if the users need to configure this as an extension setting.